tests/post_training/data/wc_reference_data.yaml

```diff
@@ -39,4 +39,8 @@ tinyllama_data_aware_lora_stateful_backend_OV:
 tinyllama_NF4_scale_estimation_stateful_per_channel_backend_OV:
   metric_value: 0.88663
   num_int4: 11
-  num_int8: 290
\ No newline at end of file
+  num_int8: 290
+tinyllama_awq_backup_mode_none_backend_OV:
+  metric_value: 0.85679
```

Comment:
@alexsu52 @l-bat is it what we get on developer hosts: CLX (Intel(R) Core(TM) i9-10980XE CPU @ 3.00GHz)?